src/components/LibraryContainer.tsx

```diff
@@ -36,6 +36,8 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
     searchCategories: string[] = [];
 
     timeout: number;
+    selectionIndex: number = -1;
```

Comment:
There is not a selected item by default. The expected behavior should be that when you are searching, and then press Enter, a node will be created from the 1st search result.